src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/casenotes/CaseNotesReconciliationService.kt

```diff
@@ -238,7 +243,6 @@ data class ComparisonCaseNote(
   val creationDateTime: String?,
   val dpsUsername: String? = null,
   val nomisUsernames: List<String>? = null,
-  val amendments: Set<CommonAmendmentFields>,
```

Comment:
amendments not now used in comparison since DPS can have more amendments than NOMIS due to the NOMIS text holding all the amendments.